src/pages/guides/using/idcs/idcs-events-data-stream-setup.md

```diff
@@ -0,0 +1,265 @@
+          "source": "indesign",
+          "value": {
+            "jobId": "8b524c70-6779-47f1-ac2a-ce93c23b3f5f",
+            "eventId": "70db1be2-cdde-4a00-bc41-5acd33db3cc7",
+            "state": "ASSET_DOWNLOAD_FAILED",
+            "timestamp": 1720631449367,
+            "data": {
+              "destination": "doc.indd",
+              "source": {
+                "assetId": "urn:aaid:sc:VA6C2:6b14edb7-7acd-34ad-825e-b70c3b2569f4",
+                "type": "ACP"
+              }
+            }
+          }
+        },
+        "id": "7e3a89ce-d1f5-4b55-a9fc-682e3ac535e2",
+        "source": "urn:uuid:5c8f2d48-a2a7-4d0e-bb49-aabf9a72bd1e",
+        "specversion": "1.0",
+        "type": "ASSET_DOWNLOAD_FAILED",
+        "datacontenttype": "application/json",
+        "time": "2024-07-10T17:10:49.509Z",
+        "event_id": "f60a502e-c16b-49c6-b3c1-770666a1921f",
+        "recipient_client_id": "7af7ab96045640b5a6e4724810145eb9"
+      }
+    }
+  ],
+  "_page": {
+    "last": "tuna:e54a7fa0-eb13-4ad5-964c-5d682f03a188.salmon:3279e19f-7150-4e07-a223-795fb6c6fe8e.f60a502e-c16b-49c6-b3c1-770666a1921f.0.1720631451.o5nlq_28cmu1ajdkuhxc",
+    "count": 1
+  }
+}
+
+````
+
+*Data Field Definitions:*
+
+Field | Description
+--- | ---
+event_id | Unique UUID generated per event
+id | Unique UUID generated per event
+type | Type of event used for event subscription routing
+source | Context in which an event happened
+time | Timestamp of the completion of the action
+data | Event data object
+jobID | ID of the JOB in IDCS
+timesamp | Time stamp in ISO 8601 format when event was triggered
+state | Tells what this event represent in the sequence of JOB execution
+imsOrgId | Internal Adobe organization identifier
+userId | Email ID of the user in Marketo who completed the action
+
+### Event List
+
+*Note - This is a snapshot listing of most available events.  There may be some events that don't show up or no longer exist.*
+    
+
+Event state | Description
+--- | ---
+QUEUED | The job is not yet running
+ASSETS_DOWNLOAD_STARTED | Downloading has started for assets required to run this job
+ASSET_DOWNLOAD_STARTED | Emmited for individual assets, once for each asset that is downloaded
+ASSET_SCANNING_STARTED | Scanning the downloaded asset
+ASSET_DOWNLOAD_COMPLETED | Downloading of an individual asset is comepleted 
+ASSET_SCANNING_COMPLETED| Scanning of an individual asset is comepleted 
+ASSET_DOWNLOAD_FAILED | Individual downloading of an asset has failed
+ASSET_SCANNING_FAILED | Individual scanning of an asset has failed
+ASSETS_DOWNLOAD_COMPLETED | Downloading of all assets is comepleted 
+ENGINE_PROCESSING_STARTED | The job was handed over to engine to be processed
+RUNNING | Engine is still working on the job
+ENGINE_PROCESSING_COMPLETED | Engine has completed the capability execution
+ASSETS_UPLOAD_STARTED | Capability generated asset upload started
+ASSET_UPLOAD_COMPLETED | Capability generated asset upload comepleted
```

Comment:
ASSET_UPLOAD_STARTED event is missing. Please add that event.

Reply:
Done